lib/ansible/module_utils/basic.py

```diff
@@ -14,7 +14,13 @@
 if sys.version_info < _PY_MIN:
     print(json.dumps(dict(
         failed=True,
-        msg=f"ansible-core requires a minimum of Python version {'.'.join(map(str, _PY_MIN))}. Current version: {''.join(sys.version.splitlines())}",
+        msg=(
+            "ansible-core requires a minimum of Python version "
+            "{0}. Current version: {1}"
+        ).format(
+            '.'.join(map(str, _PY_MIN)),
+            ''.join(sys.version.splitlines()),
```

Comment:
The sys.version attribute typically returns a string with additional information, such as the build date and other details. Using splitlines() on sys.version might not give you the desired version format.